lib/charted.js

```diff
@@ -9,16 +9,19 @@ module.exports = function(app, charted_root) {
   if (charted_root === undefined) { charted_root = '/charted/' }
 
   app.use(charted_root + 'get', function (req, res, next) {
-    var get_url = url.parse(req.url, true).query.url
+    var urlObj = url.parse(req.url, true)
 
-    if (!get_url) {
+    if (!urlObj.query.url) {
       res.statusCode = 400
       res.end('Bad Request: no url provided')
-      next()
```

Comment:
Was getting warnings when other code that runs on the response object later tries to modify it. "Headers cannot be set after response is sent" or something similar.

The middleware is handling and returning for all requests that gets passed to it, so it doesn't seem like we should need a call to next().